.github/workflows/build_test.yml

```diff
@@ -0,0 +1,94 @@
+    - name: Git Checkout
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+    - name: Setup Java ${{ matrix.java }}
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+        java-package: jdk # (jre, jdk, or jdk+fx) - defaults to jdk
+        architecture: x64
+    - name: maven-settings-xml-action
+      uses: whelk-io/maven-settings-xml-action@v14
+      with:
+        repositories: |
+          [{ 
+            "id": "jboss-releases",
+            "name": "jboss-releases",
+            "url": "https://repository.jboss.org/nexus/content/repositories/releases",
+            "releases": { "enabled": "true" }
+          }, 
+          {
+            "id": "jboss-snapshots-repository",
+            "name": "jboss-snapshots-repository",
+            "url": "https://repository.jboss.org/nexus/content/repositories/snapshots/",
+            "snapshots": { "enabled": "true" }         
+          },
+          {
+            "id": "redhat-ga-repository",
+            "name": "Red Hat GA repository",
+            "url": "https://maven.repository.redhat.com/ga/",
+            "releases": { "enabled": "true" },
+            "snapshots": { "enabled": "false" }         
+          },
+          {
+            "id": "redhat-ea-repository",
+            "name": "Red Hat EA repository",
+            "url": "https://maven.repository.redhat.com/earlyaccess/all/",
+            "releases": { "enabled": "true" },
+            "snapshots": { "enabled": "false" }         
+          }]
+        plugin_repositories: |
+          [{ 
+            "id": "jboss-public-repository",
+            "name": "JBoss Public",
+            "url": "https://repository.jboss.org/nexus/content/groups/public-jboss/",
+            "releases": { "enabled": "true" },
+            "snapshots": { "enabled": "false" }
+          },
+          { 
+            "id": "redhat-ga-repository",
+            "name": "Red Hat GA repository",
+            "url": "https://maven.repository.redhat.com/ga/",
+            "releases": { "enabled": "true" },
+            "snapshots": { "enabled": "false" }
+          },
+          { 
+            "id": "redhat-ea-repository",
+            "name": "Red Hat EA repository",
+            "url": "https://maven.repository.redhat.com/earlyaccess/all/",
+            "releases": { "enabled": "true" },
+            "snapshots": { "enabled": "false" }
+          }]
+    - name: Build/Compile and run unit tests
+      uses: GabrielBB/xvfb-action@v1
+      with:
+        run: mvn clean verify -U -fae -B -DskipITests=true -Dmaven.test.error.ignore=true -Dmaven.test.failure.ignore=true
+    - name: Archiving test artifacts
+      uses: actions/upload-artifact@v2
+      with: 
+        name: ${{ matrix.os }}-test-reports
+        path: |
+          *tests/*/target/surefire-reports/
+          */*tests/*/target/surefire-reports/
+          **/*.log
+        retention-days: 5
```

Comment:
Please remove it, so that users can analyse and review after a 5 days periods

Reply:
Like to default 90 days?

Reply:
no problem.

Reply:
Yes, remove it to follow KISS

Reply:
done